Hotfix v5.4.1

achou11 opened the pull request:
Changes:

- Removes the release description generation step from Bitrise since it is flaky and isn't completely necessary
- Fixes a regression in apk size for the release build. Turns out that there's no easy way to override the `abiFilters` setting in `defaultConfig` via the `buildTypes` field: https://stackoverflow.com/questions/53322169/how-to-override-defaultconfig-abifilters-in-buildtypes
    - Specifying the field in `defaultConfig` causes the `buildType`-specific field to be completely ignored, which is why the release build was including all ABIs instead of the ones specified. The solution is to remove that default config specification and patch nodejs-mobile-react-native to properly handle the case where it's not specified in more recent versions of Gradle (handling an empty collection in addition to null)
    - Preview (after running `gradlew assembleRelease`):
      
      ![image](https://user-images.githubusercontent.com/18542095/158244674-8eacf71f-8955-4d7f-add8-b901d11b285c.png)

achou11 committed:
fix: fix regression in release apk build size

achou11 commented on patches/nodejs-mobile-react-native+0.6.3.patch:
```diff
@@ -25,6 +25,15 @@ index e54f25c..178b15a 100644
+ 
+     def nativeModulesABIs = android.defaultConfig.ndk.abiFilters;
+-    if (nativeModulesABIs == null) {
++    if (!nativeModulesABIs) {
```
somewhat lazy approach as null and empty collections in groovy are both falsy http://groovy-lang.org/semantics.html#the-groovy-truth

gmaclennan replied:
Huh, well that makes it quite easy. I like the idea of "groovy truth"

achou11 committed:
chore: remove release description generation step for bitrise

gmaclennan commented on bitrise.yml:
```diff
@@ -192,25 +192,12 @@ workflows:
             - name: $BITRISE_GIT_TAG
             - draft: "no"
             - files_to_upload: $RELEASE_APK_PATH
-            - body: $RELEASE_DESCRIPTION
```
Maybe add a link to the change log here?

achou11 replied:
done 👍

gmaclennan reviewed (approved):
Looks good. When you have some time (non-urgent) it would be good to submit a PR to nodejs-mobile-react-native.

achou11 commented:
@gmaclennan yeah I created an issue there already (https://github.com/nodejs-mobile/nodejs-mobile-react-native/issues/11)

after addressing all feedback is it worth just merging this? not sure we have any more build minutes on bitrise so was gonna wait until we can discuss tomorrow

achou11 committed:
chore(release): Prepare release 5.4.1
